las/__init__.py

```diff
@@ -56,11 +56,16 @@ def g():
             except ValueError:
                 output, status = result, 'succeeded'
             
+            if status != 'failed':
+                params = {'output': output}
+            else:
+                params = {'error': {'message': f'Failed while executing transition: {json.dumps(output, indent=2)}'}}
+            
             las_client.update_transition_execution(
                 transition_id=transition_id,
                 execution_id=execution_id,
                 status=status,
-                output=output,
+                **params
```

Comment:
We always use trailing commas when splitting the argument list.
```suggestion
                **params,
```